projects/ngx-darkbox-gallery/src/lib/ngx-darkbox-gallery.component.ts

```diff
@@ -16,15 +16,24 @@ export class NgxDarkboxGalleryComponent implements OnInit, OnChanges {
   images: Image[] = [];
   imageCount: number;
   currentImageIndex: number;
+  private loadedImageNumber = 0;
```

Comment:
I would prefer "loadedImageCount" here according to the total "imageCount" used in line 17

Reply:
Done in 7789d3530e1681b7c9b65f132148606a04335081